scripts/release.js

```diff
@@ -178,7 +191,3 @@ if (release_option === '5') {
 } else {
   spawnOrFail('git', ['push origin HEAD:release -f']);
 }
-process.chdir(path.join(__dirname, '../demo/meeting/serverless'));
```

Comment:
We cant just remove this :)

Reply:
If we keep it, I am not sure which is right directory...
Since we plan to clone the meeting demo in the deploy Github workflow, can we deploy that meeting demo?

Reply:
we can just clone the chime sdk meeting demo and create a new release no?

Reply:
> If we keep it, I am not sure which is right directory...
> Since we plan to clone the meeting demo in the deploy Github workflow, can we deploy that meeting demo?

The only problem with creating a release candidate in Github Actions is that we cannot publicly expose the meeting URL. So we'd need to deploy it on a local environment so that isnt exposed.

Reply:
Thanks for your advice, updated